lib/escapeHtml.js

```diff
@@ -45,27 +45,27 @@ internals.escapeHtmlChar = function (charCode) {
 
 internals.isSafe = function (charCode) {
 
-    return (typeof internals.safeCharCodes[charCode] !== 'undefined');
+    return internals.safeCharCodes.has(charCode);
 };
 
 
-internals.namedHtml = {
-    '38': '&amp;',
-    '60': '&lt;',
-    '62': '&gt;',
-    '34': '&quot;',
-    '160': '&nbsp;',
-    '162': '&cent;',
-    '163': '&pound;',
-    '164': '&curren;',
-    '169': '&copy;',
-    '174': '&reg;'
-};
+internals.namedHtml = new Map([
```

Comment:
Simple curiosity, is there any advantage for a static object to be converted to Map?

Reply:
In general I think it is better to use `Map` for lookup tables whenever you can, and avoid overloading the meaning of `object`. This also avoids issues when users can control the lookup value, eg. passing `'toString'`.